src/main/java/com/google/ehub/data/EntertainmentItemDatastore.java

```diff
@@ -26,22 +26,27 @@
   private static final String DESCRIPTION_PROPERTY_KEY = "description";
   private static final String IMAGE_URL_PROPERTY_KEY = "imageUrl";
   private static final String RELEASE_DATE_PROPERTY_KEY = "releaseDate";
+  private static final String RELEASE_DATE_TIMESTAMP_MILLIS_PROPERTY_KEY =
+      "releaseDateTimestampMillis";
   private static final String RUNTIME_PROPERTY_KEY = "runtime";
   private static final String GENRE_PROPERTY_KEY = "genre";
   private static final String DIRECTORS_PROPERTY_KEY = "directors";
   private static final String WRITERS_PROPERTY_KEY = "writers";
   private static final String ACTORS_PROPERTY_KEY = "actors";
```

Comment:
FYI: you can make these fields package private and then access them directly in your test. 

e.g.
`static final String DESCRIPTION_PROPERTY_KEY = "description";`

This way, you dont have to duplicate the constants in your test

Reply:
I am going to keep them private the way they are because I don't want other classes in the same package from being able to access them.